.gitignore

```diff
@@ -1 +1,2 @@
 composer.lock
+.idea
```

Comment:
IMHO this line should be added in your Global GitIgnore config file
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Each developer may use a different IDE so your personal configuration should not be added here